CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+# dbt_netsuite v0.17.0
+
+This release involves **breaking changes** and will require running a **full refresh**.
+
+## Bug Fixes
+- Adjusted the materialization of the `int_netsuite2__tran_with_converted_amounts` model **from incremental to [ephemeral](https://docs.getdbt.com/docs/build/materializations#ephemeral)**. There was a previous issue with the incremental strategy that could potentially produce duplicate records in certain situations ([PR #153](https://github.com/fivetran/dbt_netsuite/pull/153)).
+   - Our team has determined that the most sensible path forward is to remove the incremental logic for this model entirely, as its performance benefits have proved marginal and an ephemeral materialization will avoid any duplication issues.
+> This is a **Breaking Change**, as `int_netsuite2__tran_with_converted_amounts` will no longer materialize in the warehouse.
+
+## Feature Updates
+- Added two fields to the `netsuite2__balance_sheet` and `netsuite2__income_statement` models ([PR #151](https://github.com/fivetran/dbt_netsuite/pull/151)):
```

Comment:
I suggest adding a reason why these were added (I took this from Jared's issue but feel free to adjust).
```suggestion
- Added two fields to the `netsuite2__balance_sheet` and `netsuite2__income_statement` models to support reporting on amounts in the functional currency alongside consolidated (`converted_amount`) results ([PR #151](https://github.com/fivetran/dbt_netsuite/pull/151)):
```